src/patterns/phone-numbers/index.njk

```diff
@@ -0,0 +1,65 @@
+---
+title: Phone numbers
+group: Ask users for...
+---
+{% from "views/partials/example/_macro.njk" import patternlibExample %}
+{% from "components/panel/_macro.njk" import onsPanel %}
+{% from "components/external-link/_macro.njk" import onsExternalLink %}
+{% from "components/lists/_macro.njk" import onsList %}
+
+How to ask users for a phone number.
+
+{{
+    patternlibExample({"path": "components/input/examples/telephone/index.njk"})
+}}
+
+## When to use this pattern
+Only ask users for a phone number when your service has a genuine need for this information. For example, to send an access code by text message. 
+
+## How to use this pattern
+You should always explain why you need the user's phone number, letting them know if you will contact them and when.
+
+Use an [input component](/components/input) and set the input type to `tel` to bring up the correct keypad on touch devices.
+
+### Explain why you need the phone number
+
+Use the `label__description` with the [input component](/components/input) to reassure users why you need a phone number and what it will be used for.
+
+### Use autocomplete 
+Use the `autocomplete` attribute when you are asking for a phone number as it helps the user fill out a form more quickly.
+
+To do this, set the autocomplete attribute on the input field to `tel`.
+
+You will need to use the autocomplete attribute to meet [WCAG 2.1 AA standards for accessibility](https://www.w3.org/TR/WCAG21/#identify-input-purpose).
+
+### Error messages
+{{
+    patternlibExample({"path": "patterns/phone-numbers/examples/phone-number-error/index.njk"})
+}}
+
+Use the [correct errors pattern](https://ons-design-system.netlify.app/patterns/error-validation) and show the error details above the input phone number area.
+
+#### If nothing is entered and the answer is required
+Use “Enter [the required type of phone number]”, for example, “Enter a UK mobile number”.
+
+#### If a phone number is incorrect
+Use “Enter a UK mobile number in a valid format, for example, 07700 912345 or +44 7700 912345”.
```

Comment:
What should the user say if they need to ask for a different type of phone number? Would the examples need to change?

Reply:
Yes they would, but not sure we need to have an example for every type of number - could get very busy

Reply:
yes they would but do we need examples of them all. Could end up being a really long list

Reply:
So maybe we add some guidance like: 

> "Make sure any examples phone numbers in the error message are suitable for the type of number you are asking for.  [Ofcom maintains a list of numbers](https://www.ofcom.org.uk/phones-telecoms-and-internet/information-for-industry/numbering/numbers-for-drama) reserved for use in media that you can use.

Reply:
Can we put conditions on the use of 'OTHER' numbers? So if we are only asking for a UK number that stands, but if we are open to any number can this be turned on with a different message?

Reply:
> Do we want to have any restrictions on what the user can actually enter into the input? As in not allow them to enter letters for example or put a limit on the amount of characters that the user can input?

The validation is already built into it but we may need explain this more in the documentation. With regards to the character count, research has shown that we need to allow the user to enter phone numbers in a format that suits them, e.g: spaces so we shouldn't restrict the length.